lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -190,7 +190,7 @@ def test_get_accessible_discussion_modules(self, modulestore_type, expected_disc
         self.assertNotIn(test_discussion.location, self.store.get_orphans(course.id))
 
         # Assert that there is only one discussion module in the course at the moment.
```

Comment:
Here's an example of "discussion modules" that should be renamed to "discussion xblocks".  See longer discussion on renaming here: https://github.com/edx/edx-platform/pull/12582/files#r67697669.